### PR TITLE
Kernel Explorer: More Improvements

### PR DESCRIPTION
**Pull request by elad335**
* Protocol information of lv2 objects has been added.
* Add Mutex ID of conditional variables, same goes for LW mutex and LW cond.
* Shared memory section has been renamed to "Shared Memory" instead of "Memory" as I may add sys_memory allocations info in the future and try to avoid ambiguity when it happens.
* Virtual Memory (from sys_vm) information has been added.
* Timers information has been added.
* Network sockets information has been added.

### CODE REVIEW

File: rpcs3/rpcs3qt/kernel_explorer.cpp [hunk 1]
```diff
@@ -169,14 +171,16 @@ void kernel_explorer::Update()
...
 		return;
 	}
 
 	static const size_t additional_size = 6;
 
 	const std::unordered_map<u32, QString> tree_item_names =
 	{
-		{ SYS_MEM_OBJECT                 , tr("Memory")},
+		{ SYS_MEM_OBJECT                 , tr("Shared Memory")},
+		{ virtual_memory                 , tr("Virtual Memory")},
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/kernel_explorer.cpp:183]: I seperated these visually with an empty line for a reason :)

Comment (Megamouse) [reply rpcs3/rpcs3qt/kernel_explorer.cpp:183]: hmm, I guess the order matters to you

File: rpcs3/rpcs3qt/kernel_explorer.cpp [hunk 2]
```diff
@@ -391,38 +395,41 @@ void kernel_explorer::Update()
...
+			add_leaf(node, qstr(fmt::format(u8"LWMutex 0x%08x: “%s”, %s,%s Owner: %s, Locks: %u, Wq: %zu", id, lv2_obj::name64(lwm.name), lwm.protocol,
 					(lwm_data.attribute & SYS_SYNC_RECURSIVE) ? " Recursive," : "", owner_str, lwm_data.recursive_count, lwm.sq.size())));
 			break;
 		}
 		case SYS_TIMER_OBJECT:
 		{
-			// auto& timer = static_cast<lv2_timer&>(obj);
-			add_leaf(node, qstr(fmt::format("Timer 0x%08x", id)));
+			auto& timer = static_cast<lv2_timer&>(obj);
+			sys_timer_information_t info; timer.get_information(info);
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/kernel_explorer.cpp:409]: two operations in one line

**Comment by elad335** [comment]
* Network sockets information has been added.

**Comment by Megamouse** [comment]
Can you do me a favor and change this in qt_utils.cpp ?
image

**Pull request merged**